pkg/utils/x509.go

```diff
@@ -0,0 +1,44 @@
+package utils
+
+import (
+	"time"
+
+	"github.com/openshift/library-go/pkg/crypto"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/apiserver/pkg/authentication/user"
+)
+
+func ComplianceOperatorRootCA(certname string, expire int) ([]byte, []byte, error) {
+	config, err := crypto.MakeSelfSignedCAConfig(certname, expire)
```

Comment:
Let's reach out to the crypto team... when using FIPS mode, is this function also forked to have usable defaults?